src/jit/emitxarch.cpp

```diff
@@ -4888,19 +4916,54 @@ void emitter::emitIns_AX_R(instruction ins, emitAttr attr, regNumber ireg, regNu
 }
 
 #if FEATURE_HW_INTRINSICS
-void emitter::emitIns_SIMD_R_R_R(instruction ins, regNumber reg, regNumber reg1, regNumber reg2, var_types simdtype)
+void emitter::emitIns_SIMD_R_R_R(instruction ins, regNumber reg, regNumber reg1, regNumber reg2, emitAttr attr)
```

Comment:
nit: now that this it taking `emitAttr`, rather than `var_types`, it would be nice to have it ordered the same as `emitins_R_R_R` (ins, attr, reg, ...)

Reply:
Hmm.. I think the different parameter-order can abvoid misusing `emitins_R_R_R` and `emitins_SIMD_R_R_R`.

Reply:
I'm not sure breaking parameter order to prevent misuse is a good enough reason. It looks like we tend to keep the same parameter order for most functions, even when just an extension of the previous. 

I don't feel strongly about this, however.

Reply:
We are already using different name prefix `emitIns_SIMD`. Would it be enough to prevent misuse?